### PR TITLE
docs: quick start docs of GreptimeCloud

### PR DESCRIPTION
**Pull request by nicecui**
The quick start documentation of GreptimeCloud.

Note: These docs won't be displayed as single pages at docs.greptime.com, and will be inserted in the Getting Started docs of GreptimeCloud. So there are no H1 titles in the content, and website developers cannot use the H1 title as tab titles when developing console. @alili 

The Getting Started docs were merged via #511 .

### CODE REVIEW

**Comment by nicecui** [comment]
@sunng87 PTAL

File: docs/greptimecloud/quick-start/influxdb.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+To quickly get started with InfluxDB line protocol, we can use Bash to collect system metrics, such as CPU and memory usage, and send it to GreptimeCloud.
+
+```shell
+curl -L https://github.com/GreptimeCloudStarters/quick-start-influxdb-line-protocol/releases/download/v1.0.0/quick-start.sh --output quick-start.sh && chmod +x quick-start.sh && ./quick-start.sh -h <host> -d <dbname> -u <username> -p <password>
+```
+
+The source code is avaliable at [Github](https://github.com/GreptimeCloudStarters/quick-start-influxdb-line-protocol).
```
Comment (sunng87) [inline docs/greptimecloud/quick-start/influxdb.md:8]:
```suggestion
The source code is avaliable on [Github](https://github.com/GreptimeCloudStarters/quick-start-influxdb-line-protocol).
```

File: docs/greptimecloud/quick-start/influxdb.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+To quickly get started with InfluxDB line protocol, we can use Bash to collect system metrics, such as CPU and memory usage, and send it to GreptimeCloud.
+
+```shell
+curl -L https://github.com/GreptimeCloudStarters/quick-start-influxdb-line-protocol/releases/download/v1.0.0/quick-start.sh --output quick-start.sh && chmod +x quick-start.sh && ./quick-start.sh -h <host> -d <dbname> -u <username> -p <password>
```
Comment (sunng87) [inline docs/greptimecloud/quick-start/influxdb.md:5]: curl -L https://github.com/GreptimeCloudStarters/quick-start-influxdb-line-protocol/releases/download/v1.0.0/quick-start.sh | bash -s -- -h <host> -d <dbname> -u <username> -p <password>

File: docs/greptimecloud/quick-start/java.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+Use the following command collects JVM runtime metrics, such as CPU and memory usage, and sends it to GreptimeCloud. Once the metrics are sent successfully, they can be viewed on the GreptimeCloud dashboard.
+
+```shell
+curl -L https://github.com/GreptimeCloudStarters/quick-start-java/releases/download/v0.1.1/quick-start-java-0.1.1-SNAPSHOT-all.jar --output quick-start.jar && java -jar quick-start.jar -h <host> -db <dbname> -u <username> -p <password>
+```
+
+The command line is supported by a quick start demo based on OTLP/http. The source code is available on [Github](https://github.com/GreptimeCloudStarters/quick-start-java).
```
Comment (sunng87) [inline docs/greptimecloud/quick-start/java.md:8]: Update this and next wording according to previous go demo

File: docs/greptimecloud/quick-start/python.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+Run the following command with Python 3.10+ to collect system metric data, such as CPU and memory usage, and sends it to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
+
+```shell
+pipx run --no-cache greptime-cloud-quick-start -host <host> -db <dbname> -u <username> -p <password>
+```
+
+The command line is supported by a quick start demo based on OTLP/http. The source code is available on [Github](https://github.com/GreptimeCloudStarters/quick-start-python).
```
Comment (sunng87) [inline docs/greptimecloud/quick-start/python.md:8]: Consider adding a minor desc about `pipx` as it doesn't come with python distribution by default?

**Review by sunng87 — APPROVED** [review]

File: docs/greptimecloud/getting-started/quick-start/java.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+
+Use the following command collects JVM runtime metrics, such as CPU and memory usage, and sends it to GreptimeCloud. Once the metrics are sent successfully, they can be viewed on the GreptimeCloud dashboard.
```
Comment (xtang) [inline docs/greptimecloud/getting-started/quick-start/java.md:2]:
```suggestion
Use the following command to collect JVM runtime metrics, such as CPU and memory usage, and send them to GreptimeCloud. Once the metrics have been sent successfully, they can be viewed on the GreptimeCloud dashboard.
```

File: docs/greptimecloud/getting-started/quick-start/go.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+
+The following command collects system metric data, such as CPU and memory usage, and sends it to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```
Comment (xtang) [inline docs/greptimecloud/getting-started/quick-start/go.md:2]:
```suggestion
The following command collects system metric data, such as CPU and memory usage, and sends them to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```

File: docs/greptimecloud/getting-started/quick-start/mysql.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+
+To quickly get started with MySQL, we can use Bash to collect system metrics, such as CPU and memory usage, and send it to GreptimeCloud via MySQL CLI.
```
Comment (xtang) [inline docs/greptimecloud/getting-started/quick-start/mysql.md:2]:
```suggestion
To quickly get started with MySQL, we can use Bash to collect system metrics, such as CPU and memory usage, and send them to GreptimeCloud via MySQL CLI.
```

File: docs/greptimecloud/getting-started/quick-start/node.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+
+The following command collects system metric data, such as CPU and memory usage, and sends it to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```
Comment (xtang) [inline docs/greptimecloud/getting-started/quick-start/node.md:2]:
```suggestion
The following command collects system metric data, such as CPU and memory usage, and sends them to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```

File: docs/greptimecloud/getting-started/quick-start/python.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
+
+Run the following command with Python 3.10+ to collect system metric data, such as CPU and memory usage, and sends it to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```
Comment (xtang) [inline docs/greptimecloud/getting-started/quick-start/python.md:2]:
```suggestion
Run the following command with Python 3.10+ to collect system metric data, such as CPU and memory usage, and send them to GreptimeCloud. Once sent successfully, the metrics can be viewed on the GreptimeCloud dashboard.
```

**Review by xtang — APPROVED** [review]

**Pull request merged**